config/config-example.yml

```diff
@@ -10,7 +10,7 @@ load_project: ""
 # If there are multiple photo folders, set path to the folder that contains all the photo folders,
 # or provide a list of paths via the YAML list syntax (e.g., ["/path/to/photo/folder1", "/path/to/photo/folder2"])
 # If there are no photos to add (e.g., this is an existing project that already has photos in it, set to an empty string ("")
-photo_path: "/example/path/to/photo/folder"
+photo_path: "/ofo-share/emerald-point-benchmark"
```

Comment:
I feel like the primary audience of these examples should be external users instead of internal OFO collaborators. To make it clearer that external users will need to modify this line, I'd revert it back to how it was.